Models/Reviews.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Boolean
+Base = declarative_base()
+
+
+class Review(Base):
+    __tablename__ = 'reviews'
+    id = Column(Integer, primary_key=True)
+    professor = Column(String)
+    course = Column(String)
+    engagement = Column(Integer)
+    helpful = Column(Integer)
+    workload = Column(Integer)
+    quiz = Column(Integer)
+    experience = Column(Integer)
+    recommend = Column(Boolean)
+    comment = Column(String)
+    professorId = Column(Integer)
+    courseId = Column(Integer)
```

Comment:
There should definitely be more constraints here. At the very least, you probably want some "not null" and foreign key constraints, if not more.